content/src/content/jcr_root/apps/core/wcm/sandbox/components/carousel/v1/carousel/clientlibs/site/js/carousel.js

```diff
@@ -232,6 +232,17 @@
             }
         }
 
+        /**
+         * Focuses the element and prevents scrolling the element into view
+         * @param el
```

Comment:
Could we fully document the parameter here and put a clean line after the description as we do elsewhere:
`@param {HTMLElement} element Element to focus`